google-cloud-pubsub/src/test/java/com/google/cloud/pubsub/spi/v1/SubscriberImplTest.java

```diff
@@ -82,13 +82,17 @@
   static class TestReceiver implements MessageReceiver {
     private final LinkedBlockingQueue<AckReplyConsumer> outstandingMessageReplies =
         new LinkedBlockingQueue<>();
-    private AckReply ackReply = AckReply.ACK;
+    private boolean shouldAck = true;
```

Comment:
Can you add a comment here? On the surface `shouldAck = false` seems to mean "don't do anything" not "nack instead" as it is the case here.